### DISCUSSION TITLE
store other field value in metric

### DISCUSSION THREAD

**Discussion by taotao365s** [prometheus/prometheus#9997] [General]
## Proposal
**Use case. Why is this important?**

some time we want to record some key point info, such as `user_id` or `trace_id` , and store it to prometheus store,
but it is not equal `tags`(or `label`)

*“Nice to have” is not a good use case. :)*

**Comment by taotao365s** [comment]
do anyone need?

**Reply by roidelapluie** [reply]
Trace ID's are meant to be used in examplars: https://grafana.com/blog/2021/03/31/intro-to-exemplars-which-enable-grafana-tempos-distributed-tracing-at-massive-scale/
